src/test/java/io/mfj/expr/ExprTest.kt

```diff
@@ -241,6 +241,18 @@ class ExprTest {
       mapOf( "ddd" to 1.0 ),
       true )
 
+  @Test
+  fun testFloatEq() = test(
+      "1.0 = ddd",
+      mapOf( "ddd" to 1.0f ),
+      true )
+
+  @Test
+  fun testDoubleEq() = test(
+      "1.0 = ddd",
+      mapOf( "ddd" to 1.0 ),
+      true )
+
   @Test
```

Comment:
I would like to write this statement explicitly into the test suite:
"The equals operation (=) ignores precision (it uses compareTo()==0 instead of .equals())"

I suggest this test, but feel free to propose another:
```
  @Test
  fun testIgnorePrecision() = test(
      "1.0 = ddd AND 1.000 = ddd",
      mapOf( "ddd" to 1.0 ),
      true )
```

Reply:
bdf783e adds tests for this.

Reply:
looks great, thanks